gagan-maheshwari/react-admin-dashboard
# lint and ui fixes

## Description
Changes done catering to UI fixes like screen resize due to which sidebar was not hiding.
Changes for Linting errors.

## Database schema changes
None

## Tests
### Automated test cases added
None

### Files changed
src\App.js
src\components\Navbar.jsx
src\utility\component\ESCHideWrapper.jsx

### Manual test cases run
- Resize the screen
> The screen should be resized and once the screen's width is lower than 900px, the sidebar should hide.

- Run the command `npm start`
> There should be no linting errors
